src/poojap/BankAccount.java

```diff
@@ -0,0 +1,36 @@
+package poojap;
+
+public class BankAccount {
+	
+	String password;
+	String userName;
+	int bankAccountNumber;
+	
+	void setDetails() {
+		
```

Comment:
Hi @PoojaPOct2020 
You should pass the parameter into method, this looks like hard coded, please check video